lib/bolt/cli.rb

```diff
@@ -149,6 +156,17 @@ def create_option_parser(results)
                 "Parameters to a task or plan") do |params|
           results[:task_options] = parse_params(params)
         end
+
+        results[:format] = 'human'
```

Comment:
Some consistency around spacing between default/option pairs would be nice. This is the only place with a newline, and I like the newline for readability.